README.md

```diff
@@ -1 +1,44 @@
-# GitHub-Flow
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
```

Comment:
- [ ] [REQUIRED] Good job so far 👍 You should write the name of this project instead of this text(`Project Name`). If you change it, your `README` file will be descriptive 😃